scapy/contrib/automotive/can_xcp.py

```diff
@@ -0,0 +1,302 @@
+#! /usr/bin/env python
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Fabian Wiche <f.wiche@gmx.de>
+# This program is published under a GPLv2 license
+
+# scapy.contrib.description = XCP over CAN(CAN-XCP)
+# scapy.contrib.status = loads
+
+import threading
+import time
+import scapy.modules.six as six
+from scapy.config import LINUX
+from scapy.layers.can import CAN
+
+
+class Verbose():
```

Comment:
This looks unnecessary.
You could use directly the logging module, and use by default `log_runtime` from `Scapy.error`